Source/WebGPU/WebGPU/Queue.mm

```diff
@@ -691,7 +691,7 @@ static void invalidateCommandBuffers(Vector<Ref<WebGPU::CommandBuffer>>&& comman
 {
     auto device = m_device.get();
     RELEASE_ASSERT(device);
-    return *device;
+    return *device.unsafeGet();
```

Comment:
👍 thank you, we were aware of this one but I forgot to file a bug:
https://bugs.webkit.org/show_bug.cgi?id=300204